### DISCUSSION TITLE
ChatGLM3-6B模型微调用来根据题目点评代码

### DISCUSSION THREAD

**Discussion by ghost** [zai-org/ChatGLM3#1122] [Q&A] [answered]
经过GPT4生成的一些点评数据，大概五千条左右。用lora微调6B模型，感觉效果不太理想。
基本上只能根据逻辑给分，语法错误（比如少个分号，括号等）检测不出来，有时候对于 答非所问 这种情况还会给分。
我想让大家给一点建议，是应该增加一些训练数据，还是去换一个参数数量更大的模型呢？或者说有没有其他更优的方式

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
得换个更大的，比如GLM4这些新一代模型
